# Protocols
<hr style="height:3px;border:none;color:#808080;background-color:#808080;" />

## SeSaM Protocols

<br>
<h2><div id="B"><b>dATPαS Calibration for Cex and Iodine Fragmentation</div></b></h2>

1. Set up a PCR with the component listed in the table below

 || Final concentration | uL in 1 reaction |
| --- | :---: | :---: |
| 5x GoTaq buffer | 1X | 10 |
| 10 mM dNTP mix | 0.2 mM | 1 |
| 1 mM dATPaS | 0.05-0.5 | 2.5 - 25 |
| 10 uM 5&#39; biotinylated forward primer | 12.6 pmol | 1.26 |
| 10 uM Reverse primer | 12.6 pmol | 1.26 |
| pJUMP19\_Cex | 250 ng ||
| GoTaq polymerase | 2.5 unit | 0.5 |
| DIW | Total 50 uL ||

2. Set the thermal cycler following:

| | Temperature | Time | Cycle |
| --- | --- | --- | --- |
| Initial denaturation | 95°C | 2 min | 1 |
| Denaturation | 95°C | 1 min | 30 |
| Annealing | 58°C | 1 min ||
| Extension | 72°C | 1 min 40 sec (1 min/kb) ||
| Final extension | 72°C | 10 min | 1 |
| Hold | 4C | ∞ ||

3. After PCR, add 2 uL of 50 mM Iodine prepared in 100% ethanol to a 50-uL PCR
4. Incubate the reaction at 70°C for 1 hour
5. Run samples on agarose gel to analyse the fragmentation
<br>
<h2><div  id="CDE">Biotinylated DNA Generation</div></h2>

1. Set up a PCR reaction with the component listed in the table below

 || Final concentration | uL in 1 reaction |
| --- | :---: | :---: |
| 5x GoTaq buffer | 1X | 10 |
| 10 mM dNTP mix | 0.2 mM | 1 |
| 1 mM dATPaS\* | 0.1 mM | 5 |
| 10 uM Forward primer\*\* | 12.6 pmol | 1.26 |
| 10 uM Reverse primer\*\* | 12.6 pmol | 1.26 |
| pJUMP19\_Cex | 250 ng ||
| GoTaq polymerase | 2.5 unit | 0.5 |
| DIW | Total 50 uL ||

\*Only add dATPaS when generating the biotinylated forward strand to be used regarding the original SeSaM protocol
\*\*Use 5&#39;-biotinylated forward primer with non-biotinylated reverse primer for the biotinylated forward strand and use 5&#39;-biotinylated reverse primer with non-biotinylated forward primer for the biotinylated reverse strand.

2. Set the thermal cycler according to the condition suggested below

| | Temperature | Time | Cycle |
| --- | :---: | :---: | :---: |
| Initial denaturation | 95°C | 2 min | 1 |
| Denaturation | 95°C | 1 min | 30 |
| Annealing | 58°C | 1 min ||
| Extension | 72°C | 1 min 40 sec (1 min/kb) ||
| Final extension | 72°C | 10 min | 1 |
| Hold | 4°C | ∞ ||

<br>
<h2><div  id="F">Iodofragmentation</div></h2>

1. Prepare 50 mM Iodine in 100% ethanol
2. Fragment the biotinylated forward strand by adding 2 uL iodione to 50-uL reaction
3. Incubate at 70°C for 1 hour
4. Purify the fragmented PCR product with QIAGEN PCR clean-up kit

<br>
<h2><div  id="G">DNA shearing and recovery</div></h2>

1. Purify the biotinylated forward strand using QIAGEN PCR clean-up kit
2. Verify quality of the DNA with A260/A280 ratio– should be between 1.8 – 2.0
3. Dilute the DNA to 10-20 ng/uL
4. Sonicate DNA at 20% amplitude with pulse on 15 sec, pulse off 60 sec. For a smaller volume of DNA (up to 500 uL in microcentrifuge tube), sonicate for 8 cycles. For a larger volume in 15 mL tube, sonicate for 30 minutes
5. Recover diluted DNA by adding 1/10 volume of 3 M Na-Acetate pH 5.2
6. Add 2.5 volume of -20°C 100% ethanol and incubate at -20°C for at least 1 hour
7. Centrifuge DNA at 13,000 rpm for 20 minutes
8. Wash DNA pellet with room-temperature 70% ethanol and centrifuge at 13,000 rpm for 1 minute
9. Discard ethanol and repeat step 8
10. Remove ethanol as much as possible and air-dry DNA pellet at 37°C for 20 minutes, or until completely dry
11. Resuspend the pellet with nuclease-free water.

<br>
<h2><div  id="H">Biotinylated DNA Isolation- Dynabeads</div></h2>

1. Wash 50 uL of the Dynabeads™ MyOne™ Streptavidin C1 (Invitrogen) with 1 mL 2X Binding and Washing (B&amp;W) buffer (10 mM Tris-HCl pH 7.5, 1.0 mM EDTA, 2.0 M NaCl)
2. Place the tube on a magnetic rack for 2-3 minutes, then remove the solution
3. Resuspend beads with 100 uL of 2X B&amp;W buffer
4. Add 100 uL of purified PCR product
5. Incubate at room temperature, on a rotator mixer for 1 hour. Set orbital 100 rpm 3 times, reciprocal 90° 20 times and Vibro 5° 5 times.
6. Place the tube on a magnetic rack for 2-3 minutes and discard the solution
7. Wash the beads with 100 uL 1X B&amp;W buffer, mix and place the tube on magnetic rack
8. Discard the washing solution and repeat step 7
9. Melt the non-biotinylated strand by adding 100 uL of 0.1 M NaOH
10. Incubate at 37°C for 10 minutes
11. Place the tube on magnetic rack and remove the solution
12. Add 100 uL of 0.1 M NaOH to the beads and repeat step 11
13. Wash the beads with 100 uL 1X B&amp;W once and discard the solution.
14. Elute ssDNA from the beads by boiling in 50 uL of 0.1% SDS at 95°C for 10 minutes
15. Place the tube back on the magnetic rack and collect the solution
16. Remove SDS from DNA by Monarch®PCR &amp; DNA Cleanup Kit (NEB) following the oligonucleotide clean up protocol
17. Store ssDNA at -20°C until required

<br>
<h2><div  id="I">Universal base addition – Terminal Transferase</div></h2>

1. Prepare the reaction following:

| | Final concentration |
| --- | :---: |
| 10X Terminal Transferase buffer | 1X |
| 2.5 mM CoCl2 | 0.25 mM |
| Forward strand ssDNA | 650 ng |
| 10 mM dITP | 0.4 mM |
| 20 U/uL Terminal Transferase | 10 Unit |
| DIW | Total 50 uL |

2. Incubate the reaction in a thermal cycler at 37°C for 30 minutes and 70°C for 10 minutes
3. Purify the ssDNA with Monarch®PCR &amp; DNA Cleanup Kit (NEB) following the oligonucleotide clean up protocol
4. Store ssDNA at -20°C until required

<br>
<h2><div  id="J">Full-length gene synthesis</div></h2>

1. PCR using Taq polymerase following:

| | Final concentration | uL in 1 reaction |
| --- | :---: | :---: |
| 10x ThermoPol buffer | 1X | 5 |
| 10 mM dNTP mix | 0.2 mM | 1 |
| Forward strand ssDNA | 500 ng ||
| Reverse strand ssDNA | 60 ng ||
| 10 uM Non-biotinylated reverse primer | 20 pmol | 2 |
| 100% DMSO | 5% | 2.5 |
| 5 U/uL Taq polymerase | 2.5 unit | 0.5 |
| DIW | Total 50 uL ||

2. Synthesis full gene following the SeSaM protocol

| | Temperature | Time | Cycle |
| --- | :---: | :---: | :---: |
| Initial denaturation | 94°C | 3 min | 1 |
| Denaturation | 94°C | 1 min | 30 |
| Annealing | 52.7°C | 1 min ||
| Extension | 72°C | 1 min 15 sec ||
| Final extension | 72°C | 10 min | 1 |
| Hold | 4°C | ∞ ||

3. Purify the ssDNA with Monarch®PCR &amp; DNA Cleanup Kit (NEB)
4. Store DNA at -20°C until required

<br>
<h2><div  id="K">Universal base replacement</div></h2>

1. PCR using Taq polymerase following:

|| Final concentration | uL in 1 reaction |
| --- | :---: | :---: |
| 10x ThermoPol buffer | 1X | 5 |
| 10 mM dNTP mix | 0.2 mM | 1 |
| 10 uM Forward primer | 20 pmol | 2 |
| 10 uM Reverse primer | 20 pmol | 2 |
| Full-length DNA | 1 - 60 ng ||
| 100% DMSO | 5% | 2.5 |
| 5 U/uL Taq polymerase | 2.5 unit | 0.5 |
| DIW | Total 50 uL ||

2. Set the thermal cycler according to the condition suggested below

| | Temperature | Time | Cycle |
| --- | :---: | :---: | :---: |
| Initial denaturation | 94°C | 3 min | 1 |
| Denaturation | 94°C | 1 min | 30 |
| Annealing | 52.7°C | 1 min ||
| Extension | 72°C | 1 min 15 sec ||
| Final extension | 72°C | 10 min | 1 |
| Hold | 4°C | ∞ ||

3. Run PCR product on 2% agarose gel and perform gel purification
4. Store DNA at 20°C until required

## GeneORator Protocol
## General Protocols
