
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
      <link rel="icon" href="../../img/idec-wiki-logo.png">
      <meta name="generator" content="mkdocs-1.2.3, mkdocs-material-7.3.3">
    
    
      
        <title>Protocols - iDEC 2021 | Edinburgh</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.5143246d.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.3f5d1f46.min.css">
        
          
          
          <meta name="theme-color" content="#02a6f2">
        
      
    
    
    
      
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
        <style>:root{--md-text-font-family:"Roboto";--md-code-font-family:"Roboto Mono"}</style>
      
    
    
    
    
      


    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="light-blue" data-md-color-accent="blue">
  
    
    <script>function __prefix(e){return new URL("../..",location).pathname+"."+e}function __get(e,t=localStorage){return JSON.parse(t.getItem(__prefix(e)))}</script>
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#protocols" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="iDEC 2021 | Edinburgh" class="md-header__button md-logo" aria-label="iDEC 2021 | Edinburgh" data-md-component="logo">
      
  <img src="../../img/idec-wiki-logo-white-bg.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            iDEC 2021 | Edinburgh
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Protocols
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
      </label>
      
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        
<a href="https://github.com/idec2021/Edinburgh/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="iDEC 2021 | Edinburgh" class="md-nav__button md-logo" aria-label="iDEC 2021 | Edinburgh" data-md-component="logo">
      
  <img src="../../img/idec-wiki-logo-white-bg.png" alt="logo">

    </a>
    iDEC 2021 | Edinburgh
  </label>
  
    <div class="md-nav__source">
      
<a href="https://github.com/idec2021/Edinburgh/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../team/" class="md-nav__link">
        Team
      </a>
    </li>
  

    
      
      
      

  
  
  
    
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3" type="checkbox" id="__nav_3" >
      
      
      
      
        <label class="md-nav__link" for="__nav_3">
          Project
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Project" data-md-level="1">
        <label class="md-nav__title" for="__nav_3">
          <span class="md-nav__icon md-icon"></span>
          Project
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../project/background/" class="md-nav__link">
        Background
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../project/Introduction/" class="md-nav__link">
        Introduction
      </a>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3_3" type="checkbox" id="__nav_3_3" >
      
      
      
      
        <label class="md-nav__link" for="__nav_3_3">
          Results
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Results" data-md-level="2">
        <label class="md-nav__title" for="__nav_3_3">
          <span class="md-nav__icon md-icon"></span>
          Results
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../project/results/SeSaMresults/" class="md-nav__link">
        SeSaM resuts
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../../project/results/GeneORatorresults/" class="md-nav__link">
        GeneORator results
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
    
  
  
    
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_4" type="checkbox" id="__nav_4" checked>
      
      
      
      
        <label class="md-nav__link" for="__nav_4">
          Documentation
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Documentation" data-md-level="1">
        <label class="md-nav__title" for="__nav_4">
          <span class="md-nav__icon md-icon"></span>
          Documentation
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          Protocols
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        Protocols
      </a>
      
        
<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#sesam-protocols" class="md-nav__link">
    SeSaM Protocols
  </a>
  
    <nav class="md-nav" aria-label="SeSaM Protocols">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#B" class="md-nav__link">
    dATPαS Calibration for Cex and Iodine Fragmentation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#CDE" class="md-nav__link">
    Biotinylated DNA Generation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#F" class="md-nav__link">
    Iodofragmentation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#G" class="md-nav__link">
    DNA shearing and recovery
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#H" class="md-nav__link">
    Biotinylated DNA Isolation- Dynabeads
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#I" class="md-nav__link">
    Universal base addition – Terminal Transferase
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#J" class="md-nav__link">
    Full-length gene synthesis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#K" class="md-nav__link">
    Universal base replacement
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#geneorator-protocol" class="md-nav__link">
    GeneORator protocol
  </a>
  
    <nav class="md-nav" aria-label="GeneORator protocol">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#reverse-megaprimers-generation" class="md-nav__link">
    Reverse Megaprimers generation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megaprimer-pcr" class="md-nav__link">
    Megaprimer PCR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#full-length-mutant-library-purification" class="md-nav__link">
    Full-length mutant library purification
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#general-protocols" class="md-nav__link">
    General protocols
  </a>
  
    <nav class="md-nav" aria-label="General protocols">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#chemically-competent-cells" class="md-nav__link">
    Chemically competent cells
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#colony-pcr" class="md-nav__link">
    Colony PCR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#heat-shock-transformation" class="md-nav__link">
    Heat shock transformation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jump-assembly-ref-golden-gate-assembly-platform" class="md-nav__link">
    JUMP assembly ### [ref] : Golden Gate assembly platform
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#assembly-enhancement" class="md-nav__link">
    Assembly enhancement
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#high-throughput-screening-of-cex" class="md-nav__link">
    High-throughput screening of Cex
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../notebook/" class="md-nav__link">
        Lab notebook
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../useful_features/" class="md-nav__link">
        Some useful features
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#sesam-protocols" class="md-nav__link">
    SeSaM Protocols
  </a>
  
    <nav class="md-nav" aria-label="SeSaM Protocols">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#B" class="md-nav__link">
    dATPαS Calibration for Cex and Iodine Fragmentation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#CDE" class="md-nav__link">
    Biotinylated DNA Generation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#F" class="md-nav__link">
    Iodofragmentation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#G" class="md-nav__link">
    DNA shearing and recovery
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#H" class="md-nav__link">
    Biotinylated DNA Isolation- Dynabeads
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#I" class="md-nav__link">
    Universal base addition – Terminal Transferase
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#J" class="md-nav__link">
    Full-length gene synthesis
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#K" class="md-nav__link">
    Universal base replacement
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#geneorator-protocol" class="md-nav__link">
    GeneORator protocol
  </a>
  
    <nav class="md-nav" aria-label="GeneORator protocol">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#reverse-megaprimers-generation" class="md-nav__link">
    Reverse Megaprimers generation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#megaprimer-pcr" class="md-nav__link">
    Megaprimer PCR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#full-length-mutant-library-purification" class="md-nav__link">
    Full-length mutant library purification
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#general-protocols" class="md-nav__link">
    General protocols
  </a>
  
    <nav class="md-nav" aria-label="General protocols">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#chemically-competent-cells" class="md-nav__link">
    Chemically competent cells
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#colony-pcr" class="md-nav__link">
    Colony PCR
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#heat-shock-transformation" class="md-nav__link">
    Heat shock transformation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#jump-assembly-ref-golden-gate-assembly-platform" class="md-nav__link">
    JUMP assembly ### [ref] : Golden Gate assembly platform
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#assembly-enhancement" class="md-nav__link">
    Assembly enhancement
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#high-throughput-screening-of-cex" class="md-nav__link">
    High-throughput screening of Cex
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
                
                  <a href="https://github.com/idec2021/Edinburgh/blob/main/docs/documentation/protocols.md" title="Edit this page" class="md-content__button md-icon">
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"/></svg>
                  </a>
                
                
                <h1 id="protocols"><strong>Protocols</strong></h1>
<hr style="height:3px;border:none;color:#808080;background-color:#808080;" />

<h2 id="sesam-protocols"><strong>SeSaM Protocols</strong></h2>
<hr style="height:1px;border:none;color:#808080;background-color:#808080;" />

<p><em>Click on the flow chart to view the protocol</em></p>
<p><html></p>
<script src="https://cdn.rawgit.com/knsv/mermaid/6.0.0/dist/mermaid.min.js"></script>
<p><link href="https://cdn.rawgit.com/knsv/mermaid/6.0.0/dist/mermaid.css" rel="stylesheet" /></p>
<div class='mermaid'>
  graph TD; 
  A[<b>Cex DNA Template</b>] -->|FB Generation| B(dATPaS Calibration);
  A --> |Modified FB Generation| C(Biotinylated DNA Generation);
  B --> E(Biotinylated DNA Generation);
  E --> F(Iodofragmentation);
  C --> G(DNA Shearing and Recovery);
  A -->|RB Generation| D(Biotinylated DNA Generation);
  F --> H(Biotinylated DNA Isolation);
  G --> H;
  D --> H;
  H --> |Forward strand DNA| I(Universal Base Addition);
  H --> |Reverse strand DNA| J(Full-length Gene Synthesis);
  I --> J;
  J --> K[<b>Universal Base Replacement</b>];

  click B "#B" "View Protocol";
  click C "#CDE" "View Protocol";
  click D "#CDE" "View Protocol";
  click E "#CDE" "View Protocol";
  click F "#F" "View Protocol";
  click G "#G" "View Protocol";
  click H "#H" "View Protocol";
  click I "#I" "View Protocol";
  click J "#J" "View Protocol";
  click K "#K" "View Protocol";


  style A fill: #F4D291, stroke:#35455D
  style B fill: #FAEBBD, stroke:#35455D
  style C fill: #F0B1A0, stroke:#35455D
  style D fill: #F0B1A0, stroke:#35455D
  style E fill: #F0B1A0, stroke:#35455D
  style F fill: #CAF1DE, stroke:#35455D
  style G fill: #E0D1EF, stroke:#35455D
  style H fill: #B5E2EA, stroke:#35455D
  style I fill: #FFD5B9, stroke:#35455D
  style J fill: #F5C3D1, stroke:#35455D
  style K fill: #F0E1B1, stroke:#35455D
</div>

<p></html></p>
<p><br></p>
<h3 id="B"><strong>dATPαS Calibration for Cex and Iodine Fragmentation</strong></h3>
<ol>
<li>Set up a PCR with the component listed in the table below</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final concentration</th>
<th align="center">uL in 1 reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>5x GoTaq buffer</td>
<td align="center">1X</td>
<td align="center">10</td>
</tr>
<tr>
<td>10 mM dNTP mix</td>
<td align="center">0.2 mM</td>
<td align="center">1</td>
</tr>
<tr>
<td>1 mM dATPaS</td>
<td align="center">0.05-0.5</td>
<td align="center">2.5 - 25</td>
</tr>
<tr>
<td>10 uM 5&#39; biotinylated forward primer</td>
<td align="center">12.6 pmol</td>
<td align="center">1.26</td>
</tr>
<tr>
<td>10 uM Reverse primer</td>
<td align="center">12.6 pmol</td>
<td align="center">1.26</td>
</tr>
<tr>
<td>pJUMP19_Cex</td>
<td align="center">250 ng</td>
<td align="center"></td>
</tr>
<tr>
<td>GoTaq polymerase</td>
<td align="center">2.5 unit</td>
<td align="center">0.5</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total 50 uL</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<ol>
<li>Set the thermal cycler following:</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th>Temperature</th>
<th>Time</th>
<th>Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>95°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>1 min</td>
<td>30</td>
</tr>
<tr>
<td>Annealing</td>
<td>58°C</td>
<td>1 min</td>
<td></td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min 40 sec (1 min/kb)</td>
<td></td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>∞</td>
<td></td>
</tr>
</tbody>
</table>
<ol>
<li>After PCR, add 2 uL of 50 mM Iodine prepared in 100% ethanol to a 50-uL PCR</li>
<li>Incubate the reaction at 70°C for 1 hour</li>
<li>Run samples on agarose gel to analyse the fragmentation
<br></li>
</ol>
<h3 id="CDE"><strong>Biotinylated DNA Generation</strong></h3>
<ol>
<li>Set up a PCR reaction with the component listed in the table below</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final concentration</th>
<th align="center">uL in 1 reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>5x GoTaq buffer</td>
<td align="center">1X</td>
<td align="center">10</td>
</tr>
<tr>
<td>10 mM dNTP mix</td>
<td align="center">0.2 mM</td>
<td align="center">1</td>
</tr>
<tr>
<td>1 mM dATPaS*</td>
<td align="center">0.1 mM</td>
<td align="center">5</td>
</tr>
<tr>
<td>10 uM Forward primer**</td>
<td align="center">12.6 pmol</td>
<td align="center">1.26</td>
</tr>
<tr>
<td>10 uM Reverse primer**</td>
<td align="center">12.6 pmol</td>
<td align="center">1.26</td>
</tr>
<tr>
<td>pJUMP19_Cex</td>
<td align="center">250 ng</td>
<td align="center"></td>
</tr>
<tr>
<td>GoTaq polymerase</td>
<td align="center">2.5 unit</td>
<td align="center">0.5</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total 50 uL</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<p>*Only add dATPaS when generating the biotinylated forward strand to be used regarding the original SeSaM protocol
**Use 5&#39;-biotinylated forward primer with non-biotinylated reverse primer for the biotinylated forward strand and use 5&#39;-biotinylated reverse primer with non-biotinylated forward primer for the biotinylated reverse strand.</p>
<ol>
<li>Set the thermal cycler according to the condition suggested below</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Temperature</th>
<th align="center">Time</th>
<th align="center">Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td align="center">95°C</td>
<td align="center">2 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Denaturation</td>
<td align="center">95°C</td>
<td align="center">1 min</td>
<td align="center">30</td>
</tr>
<tr>
<td>Annealing</td>
<td align="center">58°C</td>
<td align="center">1 min</td>
<td align="center"></td>
</tr>
<tr>
<td>Extension</td>
<td align="center">72°C</td>
<td align="center">1 min 40 sec (1 min/kb)</td>
<td align="center"></td>
</tr>
<tr>
<td>Final extension</td>
<td align="center">72°C</td>
<td align="center">10 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Hold</td>
<td align="center">4°C</td>
<td align="center">∞</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<p><br></p>
<h3 id="F"><strong>Iodofragmentation</strong></h3>
<ol>
<li>Prepare 50 mM Iodine in 100% ethanol</li>
<li>Fragment the biotinylated forward strand by adding 2 uL iodione to 50-uL reaction</li>
<li>Incubate at 70°C for 1 hour</li>
<li>Purify the fragmented PCR product with QIAGEN PCR clean-up kit</li>
</ol>
<p><br></p>
<h3 id="G"><strong>DNA shearing and recovery</strong></h3>
<ol>
<li>Purify the biotinylated forward strand using QIAGEN PCR clean-up kit</li>
<li>Verify quality of the DNA with A260/A280 ratio– should be between 1.8 – 2.0</li>
<li>Dilute the DNA to 10-20 ng/uL</li>
<li>Sonicate DNA at 20% amplitude with pulse on 15 sec, pulse off 60 sec. For a smaller volume of DNA (up to 500 uL in microcentrifuge tube), sonicate for 8 cycles. For a larger volume in 15 mL tube, sonicate for 30 minutes</li>
<li>Recover diluted DNA by adding 1/10 volume of 3 M Na-Acetate pH 5.2</li>
<li>Add 2.5 volume of -20°C 100% ethanol and incubate at -20°C for at least 1 hour</li>
<li>Centrifuge DNA at 13,000 rpm for 20 minutes</li>
<li>Wash DNA pellet with room-temperature 70% ethanol and centrifuge at 13,000 rpm for 1 minute</li>
<li>Discard ethanol and repeat step 8</li>
<li>Remove ethanol as much as possible and air-dry DNA pellet at 37°C for 20 minutes, or until completely dry</li>
<li>Resuspend the pellet with nuclease-free water.</li>
</ol>
<p><br></p>
<h3 id="H"><strong>Biotinylated DNA Isolation- Dynabeads</strong></h3>
<ol>
<li>Wash 50 uL of the Dynabeads™ MyOne™ Streptavidin C1 (Invitrogen) with 1 mL 2X Binding and Washing (B&amp;W) buffer (10 mM Tris-HCl pH 7.5, 1.0 mM EDTA, 2.0 M NaCl)</li>
<li>Place the tube on a magnetic rack for 2-3 minutes, then remove the solution</li>
<li>Resuspend beads with 100 uL of 2X B&amp;W buffer</li>
<li>Add 100 uL of purified PCR product</li>
<li>Incubate at room temperature, on a rotator mixer for 1 hour. Set orbital 100 rpm 3 times, reciprocal 90° 20 times and Vibro 5° 5 times.</li>
<li>Place the tube on a magnetic rack for 2-3 minutes and discard the solution</li>
<li>Wash the beads with 100 uL 1X B&amp;W buffer, mix and place the tube on magnetic rack</li>
<li>Discard the washing solution and repeat step 7</li>
<li>Melt the non-biotinylated strand by adding 100 uL of 0.1 M NaOH</li>
<li>Incubate at 37°C for 10 minutes</li>
<li>Place the tube on magnetic rack and remove the solution</li>
<li>Add 100 uL of 0.1 M NaOH to the beads and repeat step 11</li>
<li>Wash the beads with 100 uL 1X B&amp;W once and discard the solution.</li>
<li>Elute ssDNA from the beads by boiling in 50 uL of 0.1% SDS at 95°C for 10 minutes</li>
<li>Place the tube back on the magnetic rack and collect the solution</li>
<li>Remove SDS from DNA by Monarch®PCR &amp; DNA Cleanup Kit (NEB) following the oligonucleotide clean up protocol</li>
<li>Store ssDNA at -20°C until required</li>
</ol>
<p><br></p>
<h3 id="I"><strong>Universal base addition – Terminal Transferase</strong></h3>
<ol>
<li>Prepare the reaction following:</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>10X Terminal Transferase buffer</td>
<td align="center">1X</td>
</tr>
<tr>
<td>2.5 mM CoCl2</td>
<td align="center">0.25 mM</td>
</tr>
<tr>
<td>Forward strand ssDNA</td>
<td align="center">650 ng</td>
</tr>
<tr>
<td>10 mM dITP</td>
<td align="center">0.4 mM</td>
</tr>
<tr>
<td>20 U/uL Terminal Transferase</td>
<td align="center">10 Unit</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total 50 uL</td>
</tr>
</tbody>
</table>
<ol>
<li>Incubate the reaction in a thermal cycler at 37°C for 30 minutes and 70°C for 10 minutes</li>
<li>Purify the ssDNA with Monarch®PCR &amp; DNA Cleanup Kit (NEB) following the oligonucleotide clean up protocol</li>
<li>Store ssDNA at -20°C until required</li>
</ol>
<p><br></p>
<h3 id="J"><strong>Full-length gene synthesis</strong></h3>
<ol>
<li>PCR using Taq polymerase following:</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final concentration</th>
<th align="center">uL in 1 reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>10x ThermoPol buffer</td>
<td align="center">1X</td>
<td align="center">5</td>
</tr>
<tr>
<td>10 mM dNTP mix</td>
<td align="center">0.2 mM</td>
<td align="center">1</td>
</tr>
<tr>
<td>Forward strand ssDNA</td>
<td align="center">500 ng</td>
<td align="center"></td>
</tr>
<tr>
<td>Reverse strand ssDNA</td>
<td align="center">60 ng</td>
<td align="center"></td>
</tr>
<tr>
<td>10 uM Non-biotinylated reverse primer</td>
<td align="center">20 pmol</td>
<td align="center">2</td>
</tr>
<tr>
<td>100% DMSO</td>
<td align="center">5%</td>
<td align="center">2.5</td>
</tr>
<tr>
<td>5 U/uL Taq polymerase</td>
<td align="center">2.5 unit</td>
<td align="center">0.5</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total 50 uL</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<ol>
<li>Synthesis full gene following the SeSaM protocol</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Temperature</th>
<th align="center">Time</th>
<th align="center">Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td align="center">94°C</td>
<td align="center">3 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Denaturation</td>
<td align="center">94°C</td>
<td align="center">1 min</td>
<td align="center">30</td>
</tr>
<tr>
<td>Annealing</td>
<td align="center">52.7°C</td>
<td align="center">1 min</td>
<td align="center"></td>
</tr>
<tr>
<td>Extension</td>
<td align="center">72°C</td>
<td align="center">1 min 15 sec</td>
<td align="center"></td>
</tr>
<tr>
<td>Final extension</td>
<td align="center">72°C</td>
<td align="center">10 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Hold</td>
<td align="center">4°C</td>
<td align="center">∞</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<ol>
<li>Purify the ssDNA with Monarch®PCR &amp; DNA Cleanup Kit (NEB)</li>
<li>Store DNA at -20°C until required</li>
</ol>
<p><br></p>
<h3 id="K"><strong>Universal base replacement</strong></h3>
<ol>
<li>PCR using Taq polymerase following:</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final concentration</th>
<th align="center">uL in 1 reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>10x ThermoPol buffer</td>
<td align="center">1X</td>
<td align="center">5</td>
</tr>
<tr>
<td>10 mM dNTP mix</td>
<td align="center">0.2 mM</td>
<td align="center">1</td>
</tr>
<tr>
<td>10 uM Forward primer</td>
<td align="center">20 pmol</td>
<td align="center">2</td>
</tr>
<tr>
<td>10 uM Reverse primer</td>
<td align="center">20 pmol</td>
<td align="center">2</td>
</tr>
<tr>
<td>Full-length DNA</td>
<td align="center">1 - 60 ng</td>
<td align="center"></td>
</tr>
<tr>
<td>100% DMSO</td>
<td align="center">5%</td>
<td align="center">2.5</td>
</tr>
<tr>
<td>5 U/uL Taq polymerase</td>
<td align="center">2.5 unit</td>
<td align="center">0.5</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total 50 uL</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<ol>
<li>Set the thermal cycler according to the condition suggested below</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Temperature</th>
<th align="center">Time</th>
<th align="center">Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td align="center">94°C</td>
<td align="center">3 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Denaturation</td>
<td align="center">94°C</td>
<td align="center">1 min</td>
<td align="center">30</td>
</tr>
<tr>
<td>Annealing</td>
<td align="center">52.7°C</td>
<td align="center">1 min</td>
<td align="center"></td>
</tr>
<tr>
<td>Extension</td>
<td align="center">72°C</td>
<td align="center">1 min 15 sec</td>
<td align="center"></td>
</tr>
<tr>
<td>Final extension</td>
<td align="center">72°C</td>
<td align="center">10 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Hold</td>
<td align="center">4°C</td>
<td align="center">∞</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<ol>
<li>Run PCR product on 2% agarose gel and perform gel purification</li>
<li>Store DNA at 20°C until required</li>
</ol>
<p><br></p>
<h2 id="geneorator-protocol"><strong>GeneORator protocol</strong></h2>
<hr style="height:1px;border:none;color:#808080;background-color:#808080;" />
<h3 id="reverse-megaprimers-generation"><strong>Reverse Megaprimers generation</strong></h3>
<ol>
<li>Perform asymmetric PCR:</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Q5 reaction buffer</td>
<td align="center">1X</td>
</tr>
<tr>
<td>dNTP mix</td>
<td align="center">0.2 mM</td>
</tr>
<tr>
<td>Mutagenic forward primers</td>
<td align="center">25 nM</td>
</tr>
<tr>
<td>Reverse primer</td>
<td align="center">0.5 uM</td>
</tr>
<tr>
<td>pJUMP19_Cex</td>
<td align="center">200 ng</td>
</tr>
<tr>
<td>Q5 high fidelity DNA polymerase</td>
<td align="center">1 unit</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total 50 uL</td>
</tr>
</tbody>
</table>
<ol>
<li>Set the thermal cycler at:</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Temperature</th>
<th align="center">Time</th>
<th align="center">Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td align="center">98°C</td>
<td align="center">2 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Denaturation</td>
<td align="center">98°C</td>
<td align="center">30 sec</td>
<td align="center">25</td>
</tr>
<tr>
<td>Annealing</td>
<td align="center">60°C</td>
<td align="center">20 sec</td>
<td align="center"></td>
</tr>
<tr>
<td>Extension</td>
<td align="center">72°C</td>
<td align="center">40 sec</td>
<td align="center"></td>
</tr>
<tr>
<td>Hold</td>
<td align="center">4°C</td>
<td align="center">∞</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<ol>
<li>Purify PCR product with QIAGEN PCR clean-up kit</li>
</ol>
<p><br></p>
<h3 id="megaprimer-pcr"><strong>Megaprimer PCR</strong></h3>
<ol>
<li>Generate the full-length gene using the reverse megaprimers</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Q5 reaction buffer</td>
<td align="center">1X</td>
</tr>
<tr>
<td>dNTP mix</td>
<td align="center">0.2 mM</td>
</tr>
<tr>
<td>Forward primer</td>
<td align="center">0.5 uM</td>
</tr>
<tr>
<td>Mutagenic reverse megaprimer</td>
<td align="center">0.05 uM</td>
</tr>
<tr>
<td>pJUMP19_Cex</td>
<td align="center">0.5 ng</td>
</tr>
<tr>
<td>Q5 high fidelity DNA polymerase</td>
<td align="center">1 unit</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total 50 uL</td>
</tr>
</tbody>
</table>
<ol>
<li>Set the thermal cycler at:</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Temperature</th>
<th align="center">Time</th>
<th align="center">Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td align="center">98°C</td>
<td align="center">2 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Denaturation</td>
<td align="center">98°C</td>
<td align="center">30 sec</td>
<td align="center">25</td>
</tr>
<tr>
<td>Annealing</td>
<td align="center">60°C</td>
<td align="center">20 sec</td>
<td align="center"></td>
</tr>
<tr>
<td>Extension</td>
<td align="center">72°C</td>
<td align="center">40 sec</td>
<td align="center"></td>
</tr>
<tr>
<td>Hold</td>
<td align="center">4°C</td>
<td align="center">∞</td>
<td align="center"></td>
</tr>
<tr>
<td><br></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
</tbody>
</table>
<h3 id="full-length-mutant-library-purification"><strong>Full-length mutant library purification</strong></h3>
<ol>
<li>Exclude methylated DNA by DpnI digestion:</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final content in the reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>PCR product</td>
<td align="center">50 uL</td>
</tr>
<tr>
<td>DpnI</td>
<td align="center">20 U/uL</td>
</tr>
<tr>
<td>2.1 Buffer</td>
<td align="center">1X</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total volume 100 uL</td>
</tr>
</tbody>
</table>
<ol>
<li>Incubate the reaction at 37°C for 2 hours</li>
<li>Inactivate DpnI by incubating at 80°C for 10 minutes</li>
<li>Run the digestion reaction on 1% agarose gel</li>
<li>Perform gel extraction using the commercial kit</li>
</ol>
<p><br></p>
<h2 id="general-protocols"><strong>General protocols</strong></h2>
<hr style="height:1px;border:none;color:#808080;background-color:#808080;" />
<h3 id="chemically-competent-cells"><strong>Chemically competent cells</strong></h3>
<ol>
<li>Inoculate a single colony of appropriate cells into 10ml LB media and culture overnight at 37°C, 200rpm</li>
<li>Inoculate 1% of the overnight culture into a fresh LB media (100 mL)</li>
<li>Incubate at 37 °C, 200rpm until OD600 = 0.3-0.6 (approx. 2 hours)</li>
<li>Transfer to 2 x 50 mLtubes and leave on ice for 30 minutes</li>
<li>Centrifuge at 4000 x g for 5 minutes at 4 °C</li>
<li>Gently resuspend pellet in 25 mL ice cold 0.1 M MgCl2 and keep on ice for 30 minutes</li>
<li>Centrifuge at 4000 xg, 5 min for 5 minutes at 4 °C</li>
<li>Gently resuspend pellet in 25 mL ice-cold 0.1 M CaCl2 and incubate on ice for 30 minutes</li>
<li>Centrifuge at 4000 xg, 5 min for 5 minutes at 4 °C</li>
<li>Gently resuspend pellet in 1.25 mL ice cold CaCl2 with 15% Glycerol solution</li>
<li>Aliquot 100 uL and flash freeze with liquid nitrogen.  Store at – 80 °C until required
<br></li>
</ol>
<h3 id="colony-pcr"><strong>Colony PCR</strong></h3>
<ol>
<li>Prepare 50 uL of sterile water in a PCR tubes</li>
<li>Pick a single colony and resuspend in 50 uL water, then draw 5 uL out into a new tube</li>
<li>Freeze 5 uL cell suspension at -20°C for at least 10 minutes</li>
<li>Prepare the master mix following the table below</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final concentration</th>
<th align="center">uL in 1 reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>5x GoTaq buffer</td>
<td align="center">1X</td>
<td align="center">4</td>
</tr>
<tr>
<td>10 mM dNTP mix</td>
<td align="center">0.2 mM</td>
<td align="center">0.5</td>
</tr>
<tr>
<td>10 uM PS1 (forward primer)</td>
<td align="center">0.1 – 1 uM</td>
<td align="center">2</td>
</tr>
<tr>
<td>10 uM PS2 (reverse primer)</td>
<td align="center">0.1 – 1 uM</td>
<td align="center">2</td>
</tr>
<tr>
<td>Cell suspension</td>
<td align="center">5 uL</td>
<td align="center">5</td>
</tr>
<tr>
<td>GoTaq polymerase</td>
<td align="center">1.25 unit</td>
<td align="center">0.1</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total 20 uL</td>
<td align="center">6.5</td>
</tr>
</tbody>
</table>
<ol>
<li>Mix the master mix with 5 uL cell suspension (DNA template) and PCR according to the condition in the table below</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Temperature</th>
<th align="center">Time</th>
<th align="center">Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td align="center">95°C</td>
<td align="center">2 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Denaturation</td>
<td align="center">95°C</td>
<td align="center">1 min</td>
<td align="center">25</td>
</tr>
<tr>
<td>Annealing</td>
<td align="center">58°C</td>
<td align="center">1 min</td>
<td align="center"></td>
</tr>
<tr>
<td>Extension</td>
<td align="center">72°C</td>
<td align="center">1 min 30 sec (1 min/kb)</td>
<td align="center"></td>
</tr>
<tr>
<td>Final extension</td>
<td align="center">72°C</td>
<td align="center">5 min</td>
<td align="center">1</td>
</tr>
<tr>
<td>Hold</td>
<td align="center">4°C</td>
<td align="center">∞</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<ol>
<li>Run the PCR product on 1% agarose gel with 100 V for 30 minutes and visualise the result under UV exposition
<br></li>
</ol>
<h3 id="heat-shock-transformation"><strong>Heat shock transformation</strong></h3>
<ol>
<li>Defrost chemically competent cells on ice.</li>
<li>Add 2ul of plasmid DNA or 10 uL of assembly reaction to 100 uL competent cells</li>
<li>Flick the tube to mix and incubate on ice for 30minutes</li>
<li>Heat shock at 42Cfor 30 seconds</li>
<li>Incubate on ice for 2 minutes</li>
<li>Add 1 mL SOC media and recover cellsat 37C, 200 rpm for 1hour</li>
<li>Spread100 uL on an appropriate antibiotic LB/agar plate</li>
<li>Spin the rest of the cells at 4000 rpm, remove 900 uL of the media, reusupend and spread onto another LB/agar plate</li>
<li>Incubate plates at 37C overnight
<br></li>
</ol>
<h3 id="jump-assembly-ref-golden-gate-assembly-platform"><strong>JUMP assembly ### [ref] : Golden Gate assembly platform</strong></h3>
<ol>
<li>Determine the concentration of the assembly parts with Nanodrop</li>
<li>Use DNA calculator (from Promega) to calculate fmol of each part</li>
<li>Prepare the assembly parts in the concentration of 20 fmol/uL</li>
<li>Set up 20 uL JUMP assembly reaction using 1 uL of all parts (or 20 fmol)</li>
<li>Add 2 uL of 10X T4 ligase reaction buffer, 1 uL of either BsmBI (for Level0 assembly) or BsaI-HF (for Level 1 assembly), and 0.25 uL of T4 ligase</li>
<li>Add sterile water to adjust the volume up to 20 uL</li>
<li>Incubate the reaction following the tables below</li>
</ol>
<p>Level 0 assembly: BsmBI</p>
<table>
<thead>
<tr>
<th align="center">Temperature</th>
<th align="center">Time</th>
<th align="center">Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td align="center">42°C</td>
<td align="center">15 min</td>
<td align="center">1</td>
</tr>
<tr>
<td align="center">42°C</td>
<td align="center">3 min</td>
<td align="center">30</td>
</tr>
<tr>
<td align="center">16°C</td>
<td align="center">3 min</td>
<td align="center"></td>
</tr>
<tr>
<td align="center">55C</td>
<td align="center">15 min</td>
<td align="center">1</td>
</tr>
<tr>
<td align="center">80C</td>
<td align="center">5 min</td>
<td align="center">1</td>
</tr>
<tr>
<td align="center">10C</td>
<td align="center">∞</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<p>Level 1 assembly: BsaI-HF</p>
<table>
<thead>
<tr>
<th align="center">Temperature</th>
<th align="center">Time</th>
<th align="center">Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td align="center">37°C</td>
<td align="center">15 min</td>
<td align="center">1</td>
</tr>
<tr>
<td align="center">37°C</td>
<td align="center">5 min</td>
<td align="center">60</td>
</tr>
<tr>
<td align="center">16°C</td>
<td align="center">5 min</td>
<td align="center"></td>
</tr>
<tr>
<td align="center">37°C</td>
<td align="center">60 min</td>
<td align="center">1</td>
</tr>
<tr>
<td align="center">80°C</td>
<td align="center">5 min</td>
<td align="center">1</td>
</tr>
<tr>
<td align="center">10°C</td>
<td align="center">∞</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<ol>
<li>Transform the reaction directly to the competent cells
<br></li>
</ol>
<h3 id="assembly-enhancement"><strong>Assembly enhancement</strong></h3>
<ol>
<li>Incubate level 1 mutant library with BmtI endonuclease to cleave sfGFP but not Cex</li>
</ol>
<table>
<thead>
<tr>
<th></th>
<th align="center">Final content in the reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>Level 1 mutant library</td>
<td align="center">200 ng</td>
</tr>
<tr>
<td>BmtI</td>
<td align="center">2 U</td>
</tr>
<tr>
<td>rCutsmart buffer</td>
<td align="center">1X</td>
</tr>
<tr>
<td>DIW</td>
<td align="center">Total volume 10 uL</td>
</tr>
</tbody>
</table>
<ol>
<li>Incubate the reaction at 37°C for 1 hour</li>
<li>Heat inactivate enzyme at 65°C for 20 minutes</li>
<li>Directly transform 10-uL reaction to 100 uL <em>E.coli BL21</em> (DE3) competent cells</li>
<li>Plate on LB/Agar/Kan/IPTG plate for screening
<br></li>
</ol>
<h3 id="high-throughput-screening-of-cex"><strong>High-throughput screening of Cex</strong></h3>
<ol>
<li>Spread mutants on LB/agar plate with 50 ug/mL Kanamycin and 0.01 mM IPTG</li>
<li>Incubate at 37°C for 48 hours</li>
<li>Prepare the screening solution by mixing 7.5 g agarose with 50 mL 1M Sodium phosphate buffer pH 6.5 and 400 mL distilled water</li>
<li>Autoclave the screening solution</li>
<li>Let the screening solution cooled down by incubating in the water bath at 48°C</li>
<li>Add 50 mL of azo-xylan to the screening solution</li>
<li>Pour the screening solution on top of the mutant agar plate</li>
<li>Let the agarose set and incubate at 37°C for 6 hours, the visible halos indicating the working Cex enzyme
<br>
<a href="#">Back to top</a></li>
</ol>
                
              
              
                


              
            </article>
          </div>
        </div>
        
      </main>
      
        
<footer class="md-footer">
  
    <nav class="md-footer__inner md-grid" aria-label="Footer">
      
        
        <a href="../../project/results/GeneORatorresults/" class="md-footer__link md-footer__link--prev" aria-label="Previous: GeneORator results" rel="prev">
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
          </div>
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Previous
              </span>
              GeneORator results
            </div>
          </div>
        </a>
      
      
        
        <a href="../notebook/" class="md-footer__link md-footer__link--next" aria-label="Next: Lab notebook" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              Lab notebook
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
          <div class="md-footer-copyright__highlight">
            Copyright &copy; 2021 iDEC. CC-BY-4.0 | The International Directed Evolution Competition (iDEC) is a Scottish Charitable Incorporate Organisation (SCIO), registered in Scotland, with charity number：SC050826
          </div>
        
        
          Made with
          <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
            Material for MkDocs
          </a>
        
        
      </div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.sections"], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../../assets/javascripts/workers/search.8397ff9e.min.js", "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.f89c2efe.min.js"></script>
      
        <script src="../../javascripts/config.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js"></script>
      
        <script src="../../javascripts/tables.js"></script>
      
        <script src="../../mermaid.js"></script>
      
        <script src="../../mermaid.initialize()"></script>
      
    
  </body>
</html>