# **iDEC 2021 | Edinburgh**<img width="180" src="https://user-images.githubusercontent.com/92064762/136870830-c3411954-7b6e-476d-bc8c-be33a24dab1f.png" img align= "right">

<div style="text-align: left">
<h2><strong>Library Generation Method Development for the Directed Evolution of Enzymes Related to Recalcitrant Polymer Degradation.</strong></h2>
</div>

<center><iframe width="653" height="368" src="https://www.youtube.com/embed/m_jL2ujURi0?modestbranding=1&autoplay=1&controls=1&showinfo=0&rel=1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope" frameborder="0" allowfullscreen></iframe></center>

<div style="text-align: justify">
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lignocellulosic biomass is an abundant waste stream which represents a rich source of glucose monomers and is currently underutilised due to ineffective current processing technology. The Edinburgh iDEC team is therefore looking into the generation of new mutant libraries with increased enzymatic activity. Libraries were generated by two differing methodologies. Sequence Saturation Mutagenesis is a random mutagenesis method which has been successfully optimised in this study, resulting in a 5-fold increase in DNA yields, enhancing subsequent library generation. Moreover, the need for expensive phosphorothioate nucleotides has been made redundant by incorporating a novel shearing method using sonication, allowing for greater accessibility to this directed evolution technique for future studies. Additionally, a library was successfully designed using the GeneORator method, in which secondary shell active site residues were targeted with the degenerate NNK codon. Preliminary high-throughput screening suggests active variants are present, however, further sequencing and kinetic studies are required.
</div>


